jetcd-core/src/test/java/io/etcd/jetcd/WatchUnitTest.java

```diff
@@ -141,11 +143,11 @@ public void testWatcherListenOnWatcherClose() throws InterruptedException {
     CountDownLatch latch = new CountDownLatch(1);
     AtomicBoolean ref = new AtomicBoolean();
     Watch.Listener listener = Watch.listener(
-      r -> {},
-      () -> {
-        ref.set(true);
-        latch.countDown();
-      }
+        r -> {},
```

Comment:
don't add extra 2 spaces for this file. like here and other changes in this file. Then the pr looks good to me.

Reply:
Done.